pkg/cmd/cli/cmd/login/loginoptions.go

```diff
@@ -270,6 +263,12 @@ func (o *LoginOptions) gatherProjectInfo() error {
 	}
 
 	projectsList, err := oClient.Projects().List(kapi.ListOptions{})
+	// if we're running on kube (or likely kube), just set it to "default"
+	if kerrors.IsNotFound(err) {
+		fmt.Fprintf(o.Out, "No ACL filtered namespaces available.  Using \"default\".  You switch projects with '%s project <projectname>':\n\n", o.CommandName)
```

Comment:
> "No ACL filtered namespaces available" - will a user have any idea what that means?

I struggled with phrasing.  A kube user wouldn't recognized "projects" either.  Say nothing about it maybe?